types/jest/index.d.ts

```diff
@@ -119,13 +120,24 @@ declare namespace jest {
         (name: string, fn?: ProvidesCallback, timeout?: number): void;
         /** Only runs this test in the current file. */
         only: It;
+        /** Skip this test. */
         skip: It;
+        /** Notes that this async test can run concurrently. */
         concurrent: It;
     }
 
     interface Describe {
+        /** 
+         * Creates a scope for holding tests
+         * 
+         * @param {string} name The name of your scope
+         * @param {fn?} ProvidesCallback The function which provides the scope to work in
```

Comment:
ProvidesCallback is a little out of place here. Should this be `@param {Function} fn`?